uniffi_bindgen/src/bindings/swift/templates/BridgingHeaderTemplate.h

```diff
@@ -61,7 +61,7 @@ typedef struct RustCallStatus {
 
 // Callbacks for UniFFI Futures
 {%- for ffi_type in ci.iter_future_callback_params() %}
-typedef void (*UniFfiFutureCallback{{ ffi_type.canonical_name() }})(const void * _Nonnull, {{ ffi_type|header_ffi_type_name }}, RustCallStatus);
+typedef void (*UniFfiFutureCallback{{ ffi_type|ffi_type_name }})(const void * _Nonnull, {{ ffi_type|header_ffi_type_name }}, RustCallStatus);
```

Comment:
> that might break returning a callback interface in an async function.

Hrm - it's complicated to know because IIRC, no functions can *return* callback interfaces anywhere, right?

I think in this implementation I could just have this arrange to call `ffi_converter_raw()` which exists purely to kill these `_Nonnull` annotations? Which I think highlights why this PR makes sense - this consideration is 100% swift specific.

I'll mull on this some more though...

Reply:
> Hrm - it's complicated to know because IIRC, no functions can return callback interfaces anywhere, right?

Rust can't return them, but I was thinking of a callback interface method that returned a callback handle.  However, that's not currently an issue either since we don't support async callback interfaces.

Seems like this would be a good thing to fix to prepare for the future, but it's not really critical.